eos/observable-impl.hh

```diff
@@ -65,45 +65,49 @@ namespace eos
         }
     };
 
-    /* Helper functions to create ObserableEntry */
+    /* Helper functions to create ObservableEntry */
```

Comment:
Do you want to split off this file as a separate commit -> is independent of B->P l v ?!

Reply:
Nope :-)